src/backend/base/langflow/initial_setup/setup.py

```diff
@@ -529,9 +529,6 @@ def load_flows_from_directory() -> None:
     flows_path = settings_service.settings.load_flows_path
     if not flows_path:
         return
-    if not settings_service.auth_settings.AUTO_LOGIN:
-        logger.warning("AUTO_LOGIN is disabled, not loading flows from directory")
-        return
```

Comment:
Hm, the reason we added this is because if you have not logged on as the superuser already, the following check will fail: https://github.com/langflow-ai/langflow/blob/main/src/backend/base/langflow/initial_setup/setup.py#L540

iirc, it was an issue with not being able to get the current user at this point in time (and/or not knowing where to place the flows).